history-files.pl

```diff
@@ -102,13 +127,16 @@
 $xcanuck_flu=GetFluMon('xcanuck');
 
 $acis60_v_flu=GetFluMon('acis60-v');
-$han_v_flu=GetFluMon('han-v');
+
+$aciscdp_v_flu=GetFluMon('aciscdp-v');
+
+$acisway_v_flu=GetFluMon('acisway');
+$ishmael_v_flu=GetFluMon('ishmael');
 
 $colossus_v_flu=GetFluMon('colossus-v');
-$acisocc_v_flu=GetFluMon('acisocc-v');
+#$han_v_flu=GetFluMon('han-v');
```

Comment:
Do you want these definitions here for luke-v and han-v?  I don't think they're needed now, right?

Reply:
Note that it is commented out. Keeping the line in there to get flumon file targets in case it's needed in the future.